subcmd/vault/encrypt/v1/v1_test.go

```diff
@@ -152,6 +156,20 @@ func (tester *encrypterTester) run(t *testing.T, tests map[string]testInput) err
 	return nil
 }
 
+func alreadyEncryptErrorTestRun(t *testing.T, tester *encrypterTester, tests map[string]testInput) error {
+	for key, _ := range tests {
+		err := tester.encrypter.Encrypt("", fmt.Sprintf("%s/%v", tester.COPY_TEST_DIR, key))
+		if err != nil {
```

Comment:
How about excluding this error handling?
Since test cases is clear, it is considered that no error will occur.

Reply:
Sounds like a good idea.
Once we make another issue, and I think I will fix it.

#87